src/java/us/kbase/mobu/tester/ConfigLoader.java

```diff
@@ -64,7 +64,7 @@ public ConfigLoader(Properties props, boolean testMode,
 
         if (user == null && tokenString == null) {
             throw new IllegalStateException("Error: KBase account credentials are not set in " +
-                    configPathInfo);
+                    configPathInfo + " or the KBASE_TEST_TOKEN environment variable");
```

Comment:
probably should make KBTT a static var

Reply:
Are you talking about the tokenString var? Wouldn't this same thing apply to almost all of the other strings declared in this method like password, testPrefix, user, authUrl, etc...?

Reply:
I mean `KBASE_TEST_TOKEN` is probably present as a string all over the place and it should probably be a static variable somewhere, and the places where its used should use that static var instead.  Worst case put it in a `Constants` class or something like that.

I'm sure there's plenty of other cases like this that duplicate strings, but this is the one that shows up in the diff so I'm mentioning it :P

Reply:
KBASE_TEST_TOKEN is a shell environment variable though, isn't it? If you are talking about the text literal in the code on line 58, that seems to be the only place it occurs in Java code:
https://github.com/kbase/kb_sdk/search?q=KBASE_TEST_TOKEN

Reply:
How does the code get the token from the environment if KBTT is only used in an error message?

Reply:
https://github.com/kbase/kb_sdk/blob/8fc4c6041cd27518476e96b928cc02c290063d73/src/java/us/kbase/mobu/tester/ConfigLoader.java#L58

Reply:
One other spot in the same file

Reply:
My bad, I should have been more clear: it is only used to fetch the env var once in the Java code across the entire repo, on line 58 as I mentioned. Otherwise it is just as part of output string and not used anywhere else in code. So we're really only talking about a single instance of KBASE_TEST_TOKEN (it is not all over the place as originally feared). The only place a static var declaration would apply would be in this file, and only for that one time that it is referenced.

Reply:
There's 3 total places the string literal occurs in the code

Reply:
The location we're talking about now doesn't show up in search

Reply:
We're spending a lot of time discussing a pretty trivial change...

Reply:
Now at least we know it is a trivial change, and not a sprawling one. I only see only one place that KBASE_TEST_TOKEN is evaluated, and the concern that it is all over the place and needs to be a static declaration doesn't apply.

Reply:
I don't see a strong case for making it a static declaration, but if AJ feels like making the change, I'm sure nobody would object!

Reply:
This doesn't seem to me like a worthwhile change to make, given where and how the string is used in that file. As you say, a lot of time is being spent discussing a trivial change; if it means a lot to you, I'm happy to review a PR about it.

Reply:
Duplicate string literals is a pretty standard code smell: 
http://rc3.org/2006/10/11/use-contants-instead-of-string-literals/
https://rules.sonarsource.com/java/RSPEC-1192
https://sqlenlight.com/help_19/html/c04d63ab-e82f-41e4-b55f-9a7852a25211.htm

Reply:
> Now at least we know it is a trivial change, and not a sprawling one.
Even if there were dozens of instances of the string it'd still be a trivial change.